### ISSUE TITLE
[🐞Bug Report] When `feat_bias` is not one, the standard deviation used in inference is incorrect.

### ISSUE THREAD

**Issue by CoderChen01** [EricGuo5513/momask-codes#109]
Dear Authors,

I believe I have identified **a potential bug**🐞, assuming my observations are correct.

In your `train_vq.py`, you pass the `mean` and `std` directly into the `dataset class`. During the initial setup of the `training set`, you apply the `feat_bias` transformation to the `std` and then **save it.**

**_The issue is that when you pass in the `std` for the `validation set`, it gets altered again by applying the `feat_bias` transformation, and this modified value is saved. This results in the final saved std being divided by `the square of feat_bias`. Consequently, this leads to inconsistencies between the z norm parameters during inference and training._**

You should pass the dataset class using `np.copy(std)` and `np.copy(mean)`.

Best regards,
Jorji

**Comment by CoderChen01** [comment]
Duplicate Issue: #103
